priv/gettext/ru/LC_MESSAGES/apns.po

```diff
@@ -180,12 +180,17 @@ msgstr "Приходи на вечеринку к 19:00, будет классн
 msgid "Come to the party at 20:00, it will be 🔥"
 msgstr "Приходи на вечеринку к 20:00, будет классно 🔥"
 
-#, elixir-format, fuzzy
-#: lib/t/push_notifications/apns.ex:250
+#, elixir-format
+#: lib/t/push_notifications/apns.ex:250 lib/t/push_notifications/apns.ex:302
 msgid "Since LIVE starts soon 🔥"
 msgstr "Since LIVE скоро начнется 🔥"
 
 #, elixir-format
-#: lib/t/push_notifications/apns.ex:241
+#: lib/t/push_notifications/apns.ex:241 lib/t/push_notifications/apns.ex:293
 msgid "Since LIVE today 🥳"
 msgstr "Since LIVE уже сегодня 🥳"
+
+#, elixir-format
+#: lib/t/push_notifications/apns.ex:312
+msgid "Only you and other new users are invited 🎉"
+msgstr "Приглашены все новички 🎉"
```

Comment:
Translations and stuff, good enough?

Reply:
Lily will refactor if needed ;)

Reply:
@akhrail1996 I think this is what's left